sklearn/datasets/base.py

```diff
@@ -573,18 +579,21 @@ def load_diabetes(return_X_y=False):
     -------
     data : Bunch
         Dictionary-like object, the interesting attributes are:
-        'data', the data to learn and 'target', the regression target for each
-        sample.
+        'data', the data to learn, 'target', the regression target for each
+        sample, 'data_filename' (added in version 0.20), the physical location
+        of diabetes data csv dataset, and 'target_filename' (added in
+        version 0.20), the physical location of diabetes targets csv datataset.
```

Comment:
I am not in favor of  `(added in version 0.20)` in the narrative.

I would add a directive on the bottom of data

```python
        the physical location of iris csv dataset.

        .. versionadded:: 0.20
           The ``filename`` attribute.
```